### ISSUE TITLE
Allow DraftJS ^0.11.0 as a peer dependency

### ISSUE THREAD

**Issue by erikt9** [thibaudcolas/draftjs-filters#77]
I haven't experienced any problems using drafts-filters with the 0.11 draftjs series.  However, npm will complain about missing peer dependencies if I'm including this current version of draftjs since this project's `package.json` has `"draft-js": "^0.10.4"` as a peer dependency.

**Comment by thibaudcolas** [comment]
Hey @erikt9, thanks for the report. I haven’t had time to test this myself but I’ve reviewed the CHANGELOG for 0.11 and I don’t think I had particular concerns with it – so if you’re ok to make a PR I’m happy to merge it, if all of the project’s unit tests pass with Draft.js 0.11.

This is what I’d expect the new target to be:

```json
  "peerDependencies": {
    "draft-js": "^0.10.4 || ^0.11.0"
  },
```

If you’re interested in contributing further, I’d like to also have Travis CI run tests for both Draft.js 0.10 and 0.11 with a test matrix – but that’s a bit more work so this can wait.

**Comment by thibaudcolas** [comment]
:tada: This issue is fixed in v2.2.4, available on npm: [draftjs-filters@2.2.4](https://www.npmjs.com/package/draftjs-filters).

<p align="right">
Generated by :package::rocket: <a href="https://github.com/semantic-release/semantic-release">semantic-release</a>
</p>
